library-aux/src/scala/AnyKind.scala

```diff
@@ -0,0 +1,7 @@
+package scala
+
+/** The super-type of all types.
```

Comment:
```suggestion
/** AnyKind plays a special role in Scala's subtype system:
 * It is a supertype of all other types no matter what their kind is.
```

Reply:
This docstring is copied from `Definitions.scala`:
https://github.com/scala/scala3/blob/010ed5ab1f009e015b3e5436cea5cb1ac2f13340/compiler/src/dotty/tools/dotc/core/Definitions.scala#L2453-L2458
It is also source of truth for scaladoc generation. I think those comments should be kept in sync.
Should I change it in `Definitions.scala` as well?

Reply:
Yes please!